arch/arm/boot/dts/qcom-apq8064-lge-nexus4-mako.dts

```diff
@@ -0,0 +1,395 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
https://spdx.org/licenses/GPL-2.0-only.html (GPL-2.0 indetifier is deprecated)

Reply:
Thanks, fixed it